File: BOJ/10039_평균점수/java/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        int count = 5;
        int totalScore = 0;
        // 5명의 점수를 입력을 받는다

        Scanner put = new Scanner(System.in);
        int score;
        for(int i=0; i<count; i++){
            // 모두 더한다 (단, 40점 미만은 40점으로 계산한다)
            score = Integer.parseInt(put.next());
            if(score < 40) {
                score = 40;
            }
            totalScore += score;
        }
        // 평균 값을 구한다
        int averageScore = totalScore/count;
        System.out.println(averageScore);
        return;
    }
}

File: BOJ/10039_평균점수/go/main.go
package main

import "fmt"

func main() {
	var totalScore int = 0
	var count int = 5
	var score int
	for i := 1; i <= count; i++ {
		// 5명의 점수를 입력 받는다
		fmt.Scan(&score)
		// 모두 더한다. (단 40점 미만은 40점으로 한다.)
		if score < 40 {
			score = 40
		}
		totalScore += score
	}
	// 평균을 구한다.
	var averageScore int = totalScore/count
	fmt.Print(averageScore)
}
